pkg/operator/ceph/controller/spec.go

```diff
@@ -339,7 +338,8 @@ func CheckPodMemory(resources v1.ResourceRequirements, cephPodMinimumMemory uint
 	if !podMemoryLimit.IsZero() {
 		// This means LIMIT and REQUEST are either identical or different but still we use LIMIT as a reference
 		if uint64(podMemoryLimit.Value()) < display.MbTob(cephPodMinimumMemory) {
-			return errors.Errorf(errorMessage, display.BToMb(uint64(podMemoryLimit.Value())), cephPodMinimumMemory)
+			// allow the configuration if less than the min, but pring a warning
```

Comment:
```suggestion
			// allow the configuration if less than the min, but print a warning
```

Reply:
Should we return as well? Especially if we enter the Request condition?

Reply:
I thought it was still good to check for the other error condition with the limit being less than the request.